lib/code_fund_web/controllers/api/ad_serve_controller.ex

```diff
@@ -214,7 +212,6 @@ defmodule CodeFundWeb.API.AdServeController do
 
   defp error_map(reason_message, pixel_url \\ "") do
     %{
-      image: "",
```

Comment:
Do you need to add large_image_url and small_image_url here? as it appears on the regular json response?